modules/logging-architecture-overview.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+//
+
+:_content-type: CONCEPT
+[id="logging-architecture-overview_{context}"]
+= Logging architecture
+
+The {logging} contains 4 logical components:
+
+* `Collector` - Reads container log data from each node.
+
+* `Forwarder` -	Forwards log data to configured outputs.
+
+* `Store` - Stores log data for analysis; the default output for the forwarder.
+
+* `Visualization` - GUI to search, query, and view stored logs.
+
+These components are managed by Operators, and Custom Resource (CR) YAML files.
+
+include::snippets/logging-log-types-snip.adoc[]
+
+Either collector uses the following sources:
+
+* journald for all system logs
+* `/var/log/pods/\*.log` for all container logs
+
+If you configure your log collector to collect audit logs, it gets them from `/var/log/audit/audit.log`. Audit logs from the Kubernetes API server are stored in the `/var/log/kube-apiserver/audit.log` file. Audit logs from the OpenShift API server are stored in the `/var/log/openshift-apiserver/audit.log` file. OVN logs are collected if available.
+
+The logging collector is a daemon set that deploys pods to each {product-title} node. System and infrastructure logs are generated by journald log messages from the operating system, the container runtime, and {product-title}. Node logs are generated by the node filesystem. Container logs are generated by the CRI-O container engine. The collector collects the logs from these sources and forwards them internally or externally as configured in the `ClusterLogForwarder` custom resource.
```

Comment:
"Container logs" is the most accurate term, since each log stream comes from the stdout/stderr of a single container. "Pod logs" is a commonly-used term, since k8s users tend to think more in terms of pods than containers and if  you're using `oc logs` you need to specify a Pod to get to the containers.
Maybe: "Container logs are generated by containers running in pods running on the cluster. Each container generates a separate log stream"